nengo/networks/tests/test_product.py

```diff
@@ -35,3 +35,100 @@ def test_sine_waves(Simulator, plt, seed):
     plt.xlim(right=t[-1])
 
     assert rmse(AB[:len(offset), :], sim.data[p][offset, :]) < 0.2
+
+
+class HilbertCurve(object):
```

Comment:
Yeah, any suggestion where to put it? `nengo.utils.weird_things_that_do_not_belong_anywhere_else`? :wink: 
Maybe `nengo.utils.functions`?